Python/compile.c

```diff
@@ -2587,7 +2620,7 @@ compiler_for(struct compiler *c, stmt_ty s)
     if (start == NULL || end == NULL || cleanup == NULL)
         return 0;
 
-    if (!compiler_push_fblock(c, FOR_LOOP, start, end))
+    if (!compiler_push_fblock(c, FOR_LOOP, start, end, NULL))
```

Comment:
If you're up for it, would you mind fixing these `if` blocks to use `{}`? Otherwise the style is shifting and I would rather promote the new style than the old one.